src/platform/ESP32/BLEManagerImpl.h

```diff
@@ -217,6 +217,7 @@ class BLEManagerImpl final : public BLEManager,
     void HandleDisconnect(esp_ble_gatts_cb_param_t * param);
     CHIPoBLEConState * GetConnectionState(uint16_t conId, bool allocate = false);
     bool ReleaseConnectionState(uint16_t conId);
+    CHIP_ERROR MapBTError(int btErr);
```

Comment:
Why is this required? Can't there be a single function `MapBLEError` which is used for both Bluedroid and NimBLE?

Reply:
Done. Added the single function `MapBLEError` for both.